### PR TITLE
Updated ref arch content

### PR DESCRIPTION
**Pull request by bwhaley**
Updated the reference architecture product page with the new diagram and copy. Also dropped references to GCP. Please review with an eye for typos! 👀 

Hero and subhero content:

Updated diagram:

How it works:

More detail:

Now with _hundreds_ of customers! (instead of 70):

New security features:

### CODE REVIEW

File: _data/initial-setup-how-it-works.yml [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
-    <p>You fill out an online web form to customize your Reference Architecture:</p>
+    <p>Customize your architecture and complete a few setup steps:</p>
     <ul>
-      <li>Single Account or Multi-Account</li>
-      <li>Region</li>
-      <li>End-to-end encryption (as part of HIPAA, PCI, or other compliance programs)</li>
+      <li>Create several AWS accounts (logs, security, shared, dev, stage, and prod)</li>
+      <li>Choose a region</li>
+      <li>Set up domains in DNS</li>
       <li>Run services on Docker using Kubernetes or ECS, or directly on EC2 Instances using ASGs</li>
```
Comment (josh-padnick) [inline _data/initial-setup-how-it-works.yml:8]:
```suggestion
      <li>Run services on Docker using EKS (Kubernetes) or ECS, or directly on EC2 Instances using ASGs</li>
```

File: _data/initial-setup-how-it-works.yml [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
-      <li>Kafka, ZooKeeper, ELK, MongoDB, and many other options</li>
     </ul>
 
 - title: We build your architecture
   description: |
     <p>
-      We translate your preferences into infrastructure code written in Terraform, Bash, Python, and Go. We put the
-      code into your git repos and deploy it into your AWS or GCP account(s). For AWS, this takes about one day. For
-      GCP, <a href="/contact/">Contact Us!</a>.
+      We generate he architecture using Terragrunt, Terraform, Bash, Python and Go. We deploy the resources to your
```
Comment (josh-padnick) [inline _data/initial-setup-how-it-works.yml:22]:
```suggestion
      We generate the architecture using Terragrunt, Terraform, Bash, Python and Go. We deploy the resources to your
```

File: _data/initial-setup-how-it-works.yml [hunk 3]
```diff
@@ -1,28 +1,27 @@
...
+    <p>Customize your architecture and complete a few setup steps:</p>
     <ul>
-      <li>Single Account or Multi-Account</li>
-      <li>Region</li>
-      <li>End-to-end encryption (as part of HIPAA, PCI, or other compliance programs)</li>
+      <li>Create several AWS accounts (logs, security, shared, dev, stage, and prod)</li>
+      <li>Choose a region</li>
+      <li>Set up domains in DNS</li>
       <li>Run services on Docker using Kubernetes or ECS, or directly on EC2 Instances using ASGs</li>
       <li>PostgreSQL, MySQL, SQL Server, or other relational database</li>
```
Comment (josh-padnick) [inline _data/initial-setup-how-it-works.yml:9]:
```suggestion
      <li>PostgreSQL, MySQL, SQL Server, Aurora, or other relational database</li>
```

**Review by josh-padnick — APPROVED** [review]
Thank you so much for taking the lead on this! It's always the item I never get to. Left a few small comments, but otherwise this is ready to merge!

**Comment by bwhaley** [comment]
Thanks for the review! Just incorporated all the feedback. Please approve and we'll be good to ship it!

File: _data/initial-setup-how-it-works.yml (hunk 1 above)
Comment (bwhaley) [reply _data/initial-setup-how-it-works.yml]: Committed!

File: _data/initial-setup-how-it-works.yml (hunk 3 above)
Comment (bwhaley) [reply _data/initial-setup-how-it-works.yml]: Committed!

Comment (bwhaley) [reply _data/initial-setup-how-it-works.yml]: Committed!  8c08a69

File: _data/initial-setup-how-it-works.yml (hunk 2 above)
Comment (bwhaley) [reply _data/initial-setup-how-it-works.yml]: Committed! 8c08a69

**Comment by josh-padnick** [comment]
@eak12913 @oredavids @ebeneliason Could we get a rubber stamp on this? Regrettably, I've become a contributor and can't merge myself.

**Comment by yorinasub17** [comment]
@josh-padnick if you hit approve, I believe we can still merge this, since the primary commit author for the suggestions is still Ben.

**Comment by josh-padnick** [comment]
@yorinasub17 The issue is that I made suggestions which Ben accepted, making me a contributor. I know, I hate this!

**Comment by bwhaley** [comment]
What Yori is saying is that even though I accepted your suggestions and you became a committer, you can still approve by going to Files and Approve. After approving you should see the button turn green to merge. We confirmed this behavior on the service catalog and other repos.

**Review by josh-padnick — APPROVED** [review]

**Comment by josh-padnick** [comment]
Oh my gosh, I did not know this! What a revelation.

@ebeneliason @oredavids @eak12913 Note that code committers can still go to files / approve on PRs to approve them.

**Comment by zackproser** [comment]
@josh-padnick do we tend to leave branches around for the public website? If not, probably want to delete this.

**Comment by josh-padnick** [comment]
Ah good call. Deleted now.

**Pull request merged**